web3-functions/index.ts

```diff
@@ -0,0 +1,293 @@
+import {
+  Web3Function,
+  Web3FunctionContext,
+} from "@gelatonetwork/web3-functions-sdk";
+import { BigNumber, Contract } from "ethers";
+import { arrayify, toUtf8String, formatBytes32String } from "ethers/lib/utils";
+import { WrapperBuilder } from "@redstone-finance/evm-connector";
+import * as redstone from "redstone-protocol";
+
+const ORACLE_ABI = [
+  "function updateDataFeedsValuesPartial(bytes32[]) public",
+  "function getLastUpdateDetails(bytes32) public view returns (uint256, uint256, uint256)",
+  "function getLivePricesAndTimestamp(bytes32[]) public view returns (uint256[], uint256)",
+];
+
+const MIN_DEVIATION = 0.5; // 0.5%
+const MIN_TIME_ELAPSED = 6; // 6 hours
```

Comment:
Since this is the max time we wait since last price update.
```suggestion
const MAX_TIME_ELAPSED_HOURS = 6; // 6 hours
```

Reply:
Will be addressed in separate [Issue](https://github.com/LiskHQ/lisk-contracts/issues/210)